vulnerabilities/mapping-within-struct.md

```diff
@@ -0,0 +1,21 @@
+# Deleting a mapping within a struct
```

Comment:
Can we capitalize the title to "Deleting a Mapping Within a Struct"

Reply:
Fixed in the latest commit!